### DISCUSSION TITLE
Hide markets when zoomed out

### DISCUSSION THREAD

**Discussion by TimPushkin** [peterLaurence/MapCompose#64] [Q&A] [answered]
I have a map with a lot of markers which start to overlap when the map is zoomed out. Is it possible to hide all (or some of) the markers when the map is zoomed out (when zoom value is lower than some predefined value)?

**Comment by Nohus** [comment, marked as answer]
You can just update your marker list when the zoom level changes. I have an allowed scale range on my marker class, and filter by that when the map scale changes.

**Reply by TimPushkin** [reply]
Thanks, I missed that `MapState` provides `setChangeListener()`. And I can also use `MapState.scale` inside the composable passed to `setMarker()`.
